src/modules/m_connameban.cpp

```diff
@@ -0,0 +1,56 @@
+/*
+ * InspIRCd -- Internet Relay Chat Daemon
+ *
+ *   Copyright (C) 2016 Johanna Abrahamsson <johanna-a@users.noreply.github.com>
+ *
+ * This file is part of InspIRCd.  InspIRCd is free software: you can
+ * redistribute it and/or modify it under the terms of the GNU General Public
+ * License as published by the Free Software Foundation, version 2.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more
+ * details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+
+#include "inspircd.h"
+
+class ModuleConnameBan : public Module, public Whois::EventListener
+{
+	public:
+		Version GetVersion() CXX11_OVERRIDE
+		{
+			return Version("Conname 'n' - Connection name (class) ban",VF_OPTCOMMON);
```

Comment:
The flags should be `VF_VENDOR | VF_OPTCOMMON` as it's going to be part of the official distribution.